package.json

```diff
@@ -58,13 +58,13 @@
     "popover"
   ],
   "peerDependencies": {
+    "@popperjs/core": "^2.0.0",
```

Comment:
This shouldn't be here, having it in `dependencies` is enough.

Reply:
This is common practice (see [react-redux](https://github.com/reduxjs/react-redux/blob/master/package.json#L40) or [mobx-react](https://github.com/mobxjs/mobx-react/blob/master/package.json#L44)). And now we don't need re-export things like `placements` of `strategies`. User can import all that he or she need directly from original library.